tls/s2n_security_policies.c

```diff
@@ -574,6 +582,7 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version="KMS-PQ-TLS-1-0-2020-07", .security_policy=&security_policy_kms_pq_tls_1_0_2020_07, .ecc_extension_required=0, .pq_kem_extension_required=0 },
     { .version="PQ-SIKE-TEST-TLS-1-0-2019-11", .security_policy=&security_policy_pq_sike_test_tls_1_0_2019_11, .ecc_extension_required=0, .pq_kem_extension_required=0 },
     { .version="PQ-SIKE-TEST-TLS-1-0-2020-02", .security_policy=&security_policy_pq_sike_test_tls_1_0_2020_02, .ecc_extension_required=0, .pq_kem_extension_required=0 },
+    { .version="PQ-TEST-TLS-1-0-2020-12", .security_policy=&security_policy_pq_test_tls_1_0_2020_12, .ecc_extension_required=0, .pq_kem_extension_required=0 },
```

Comment:
I don't think we need `TEST` in the name here. My understanding was that we added `TEST` to the other security policies to warn users that it's not recommended to use them normally since they'd be likely to run into performance issues since it only has SIKE which can be pretty slow. 

This preference just looks like `cipher_suites_kms_pq_tls_1_0_2020_07` with TLS 1.3 ciphers added which shouldn't have those performance issues.

Reply:
That makes sense. I've updated it.